### PR TITLE
[#141] Configurable actions in the UI

### PR DESCRIPTION
**Pull request by ponchofiesta**

### CODE REVIEW

File: kitodo-mediaserver-core/src/main/resources/config/local.yml [hunk 1]
```diff
@@ -31,3 +31,13 @@ importer:
   validationFileGrps:
   - PRESENTATION
   - FULLTEXT
+
+ui:
+  works:
+    actions:
+      CacheClear:
```
[the remark this answers is not in the document]
Comment (ponchofiesta) [reply kitodo-mediaserver-core/src/main/resources/config/local.yml]: Exactly my thoughts :) AFAIK you can not delete entries of the default.yml. I put it to the local.yml, so the user can remove the entry.

**Comment by ponchofiesta** [comment]
Yes, I thought about that too. My idea was to extend the IAction interface to provide a getLabel(). Every Action should provide it's name or a key to the messages translation file. 

Actually the Key in the local.yml is the value which is displayed and the user can choose this string by itself. Multilanguage support with a custom text would be difficult. Should this be done by the configuration like this?
```yaml
testAction:
  label:
    de_DE: Titel in deutsch
    en: English title
```

**Comment by pbroman** [comment]
I'm not so fond of the idea of having multi-language messages in the yml config. Messages should always be in messages_*.properties.
Also, it would be nicer to provide certain default actions and make it possible to deactivate them in local.yml. I don't yet have a good idea for the solution, though.

**Comment by pbroman** [comment]
Very nice!

**Pull request merged**